### PR TITLE
docs: add custom comparator and `unique` validation type tutorial

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* added custom comparator tutorial
* added `unique` validation type tutorial and `errorInfo` property tutorial

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: packages/toast-ui.grid/docs/en/sort.md [hunk 1]
```diff
@@ -101,6 +101,35 @@ const grid = new Grid({
 Currently, data synced using the `Data Source` can only be sorted individually. While we plan on implementing the functionality of sorting multiple columns at once, at the moment, it is not yet supported. 
 
 
+## Custom Comparator
+
+If you want to sort the data with your own comparator, the `comparator` option can be useful for it. 
+The structure of the Custom Comparator is same as [compareFunction](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) that runs in the javaScript `sort` function, but there is a difference that row information to be sorted can also be used as parameters. It is possible to sort the data with other column data using these parameters.
```
Comment (seonim-ryu) [inline packages/toast-ui.grid/docs/en/sort.md:107]: javaScript -> JavaScript 로 변경해야겠네요ㅎ

File: packages/toast-ui.grid/docs/en/sort.md [hunk 2]
```diff
@@ -101,6 +101,35 @@ const grid = new Grid({
...
+
+```js
+const comparator = (valueA, valueB, rowA, rowB) => {
```
Comment (seonim-ryu) [inline packages/toast-ui.grid/docs/en/sort.md:110]: `row` 값 사용한 예제도 추가되면 좋을 것 같습니다ㅎ

File: packages/toast-ui.grid/docs/en/sort.md [hunk 3]
```diff
@@ -101,6 +101,35 @@ const grid = new Grid({
...
+  columns: [
+    { name: 'alphabetA', header: 'alphabetA', comparator  },
+    // ...
+  ]
+})
+```
+
+### Note
+
+Custom Comparator feature can only be implemented in `v4.14.0` and above.
```
Comment (seonim-ryu) [inline packages/toast-ui.grid/docs/en/sort.md:131]: 별건 아니지만... Note 내용은 블록쿼트로 처리하면 어떨까요??
```markdown
> Note: Custom Comparator feature can only be implemented in `v4.14.0` and above. 
```

File: packages/toast-ui.grid/docs/en/validation.md [hunk 4]
```diff
@@ -144,6 +144,56 @@ const grid = new Grid({
...
+            meta: { customErrorCode: 'CUSTOM_ERROR_CODE' }
+          };
+        }
+      }
+    }
+  ]
+});
+```
+
+## unique Option
```
Comment (jung-han) [inline packages/toast-ui.grid/docs/en/validation.md:172]: `unique`로 하거나 Unique 여야 할 것 같습니다!

File: packages/toast-ui.grid/docs/ko/sort.md [hunk 5]
```diff
@@ -100,6 +100,33 @@ const grid = new Grid({
...
+
+```js
+const comparator = (valueA, valueB, rowA, rowB) => {
+  if (valueA.length < valueB.length) {
+    return -1;
+  }
+  if (valueA.length > valueB.length) {
+    return 1;
+  }
+  return 0;
```
Comment (jung-han) [inline packages/toast-ui.grid/docs/ko/sort.md:115]: 별 건 아니지만..
`return valueA.length - valueB.length;` 이런식으로 작성하는건 어떤가요?_?

File: packages/toast-ui.grid/docs/ko/validation.md [hunk 6]
```diff
@@ -168,6 +167,56 @@ const grid = new Grid({
...
+        }
+      }
+    }
+  ]
+});
+```
+
+## unique 옵션
+
+`v4.14.0` 이상부터 사용할 수 있는 옵션이다. `unique` 옵션을 `true`로 설정하면 셀 데이터가 컬럼에서 중복되는지 검사할 수 있다. 이 때 셀 데이터가 중복되는 경우에는 빨간색으로 표시된다.
```
Comment (jung-han) [inline packages/toast-ui.grid/docs/ko/validation.md:197]: 위에 선임*2님이 말씀해주신 것처럼 버전 안내는` >`로 작성하면 좋을 것 같습니다.

**Pull request merged**